index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" type="text/css" />
+    <title>Portfolio</title>
+  </head>
+  <body>
+    <header class="wrapper">
+      <nav class="container">
+        <div>
+          <a href="#"><img src="./assets/img/logo.png" alt="logo" /></a>
+        </div>
+        <div class="toggle">
+          <a href="#"><img src="./assets/img/toggle.png" alt="toggle" /></a>
+        </div>
+      </nav>
+    </header>
+    <section class="showcase">
+      <div class="showcase_text">
+        <h1 id="full-name">
+          Biftu <br />
+          Girma
+        </h1>
+        <p>
+          Hello! I am a software developer! I can help you build a product,
+          feature or website. Take a look at my work. <br /><br />
+          If you like what you see and have a project you need coded, don’t
+          hesitate to contact me.
+        </p>
+
+        <a class="primary-btn" href="#">Start collaboration</a>
+      </div>
+
+      <div class="showcase_img">
+        <img src="./assets/img/headline_img.png" alt="headline_img" />
+      </div>
```

Comment:
- [OPTIONAL] You added correctly the image to the page as the previous code reviewer mentioned, but the first section background image would be better added as a `background-image` CSS property to the first section instead of an `img` tag. If an image is not part of the content (aesthetic reasons) use CSS `background-image`. [Check this link](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image) for more info.